dotcom-rendering/src/web/components/LiveBlock.tsx

```diff
@@ -63,22 +63,23 @@ export const LiveBlock = ({
 			host={host}
 			pageId={pageId}
 		>
-			{block.elements.map((element, index) =>
-				renderArticleElement({
-					format,
-					element,
-					isMainMedia: false,
-					host,
-					adTargeting,
-					ajaxUrl,
-					index,
-					pageId,
-					webTitle,
-					isAdFreeUser,
-					isSensitive,
-					switches,
-				}),
-			)}
+			{block.elements.map((element, index) => (
+				<RenderArticleElement
+					key={'elementId' in element ? element.elementId : index}
```

Comment:
For simplicity is it worth just using `index` instead of relying on elementId at all?

Reply:
Or `element.elementId ?? index` ?

Reply:
Unfortunately elementId doesn't exist on the element type due to the way elements are typed in DCR, which is why the funky syntax is used!

I'm going to switch to this:
```jsx
// eslint-disable-next-line react/no-array-index-key -- This is only rendered once so we can safely use index to suppress the warning
```

Worth remembering that `index` is _not recommended_ for use as the key prop and the only reason its safe is because this is being server side rendered! We even have an eslint rule reminding us not to do it!